### PR TITLE
Sdf pointer (for sdfRequired)

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -1077,8 +1077,19 @@ The keyword "sdfRequired" is provided to apply a constraint that
...
+
+References in this array can be SDF names (JSON Pointers), or one of
+two abbreviated references:
+
+* a text string with an affordance name.
+All affordance declarations that are in the same sdfObject and that
+carry this name (there can be multiple ones, one per affordance type)
+are declared to be mandatory to be represented.
+* the Boolean value `true`.
+The affordance itself that carries the sdfRequired keyword is declared
```
Comment (akeranen) [inline sdf.md:1091]: How about 'sdfRequired true' in Object/Thing definition

Comment (cabo) [reply sdf.md]: Should be fixed now; introducing "groupings" as a way to talk about sdfObject and sdfThing in one word.

File: sdf.md [hunk 2]
```diff
@@ -1077,8 +1080,20 @@ The keyword "sdfRequired" is provided to apply a constraint that
 defines for which declarations the corresponding data are mandatory in a
 Thing modeled by the current definition.
 
-The value of "sdfRequired" is an array of name references (JSON pointers), each
-indicating one declaration that is mandatory to be represented.
+The value of "sdfRequired" is an array of references, each indicating
+one or more declarations that are mandatory to be represented.
+
+References in this array can be SDF names (JSON Pointers), or one of
+two abbreviated references:
```
Comment (akeranen) [inline sdf.md:1087]:
```suggestion
two abbreviated reference formats:
```

File: sdf.md [hunk 3]
```diff
@@ -1077,8 +1080,20 @@ The keyword "sdfRequired" is provided to apply a constraint that
...
+
+* a text string with an affordance name or grouping name.
+All affordance declarations that are in the same grouping and that
```
Comment (akeranen) [inline sdf.md:1090]: Suggest to limit this to affordances and groupings that are directly part of the grouping where the sdfRequired is 
```suggestion
All affordance declarations that are directly (i.e., not nested further in another grouping) in the same grouping and that
```

File: sdf.md [hunk 4]
```diff
@@ -1077,8 +1080,20 @@ The keyword "sdfRequired" is provided to apply a constraint that
...
+carry this name (there can be multiple ones, one per affordance type)
+are declared to be mandatory to be represented; analogously for
```
Comment (akeranen) [inline sdf.md:1092]: Would find slightly more clear
```suggestion
are declared to be mandatory to be represented. The same applies for
```

**Review by akeranen — APPROVED** [review]
Looks good with couple of suggestions

**Pull request merged**